docs/mp/jwt.adoc

```diff
@@ -25,8 +25,27 @@
 :feature-name: JWT Authentication
 :microprofile-bundle: true
 :rootdir: {docdir}/..
+:spec-version: 1.2
 include::{rootdir}/includes/mp.adoc[]
 
+== Contents
+
+- <<Overview, Overview>>
+- <<Maven Coordinates, Maven Coordinates>>
+- <<Usage, Usage>>
+- <<API, API>>
+- <<Configuration, Configuration>>
+- <<Examples, Examples>>
+- <<Additional Information, Additional Information>>
+- <<Reference, Reference>>
+
+== Overview
+
+JSON Web Tokens are an open, industry standard https://datatracker.ietf.org/doc/html/rfc7519[RFC 7519] method for representing claims securely between two parties.
+
+JSON Web Token (JWT) defines a compact and self-contained way for securely transmitting information between parties as a JSON object. With JWT Auth you can integrate security features such as single sign on into your Helidon MP applications.
```

Comment:
Remove `(JWT)`. It makes sense to introduce a shorter name in the first mention, not in second.

Reply:
fixed